src/webview/static/base.css

```diff
@@ -142,6 +142,18 @@ body {
   padding: 0 16px 16px 80px;
 }
 
+.message-muted {
```

Comment:
I wonder if `message-muted-user` would be a clearer name for this class. I keep thinking `message-muted` sounds like it could apply somehow to a topic mute, which is a pretty different-feeling use case that gets treated differently in the UI.

Reply:
This goes away in a later commit, so I'm not too inclined to change it.